test/e2e/scheduling/preemption.go

```diff
@@ -118,21 +130,37 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			if i == 0 {
 				priorityName = lowPriorityClassName
 			}
-			pods[i] = createPausePod(f, pausePodConfig{
+			pods = append(pods, createPausePod(f, pausePodConfig{
 				Name:              fmt.Sprintf("pod%d-%v", i, priorityName),
 				PriorityClassName: priorityName,
 				Resources: &v1.ResourceRequirements{
 					Requests: podRes,
 				},
-			})
+				NodeName: node.Name,
+			}))
 			framework.Logf("Created pod: %v", pods[i].Name)
 		}
+		if len(pods) < 2 {
+			framework.Skipf("We need at least two pods to be created but" +
```

Comment:
I think this should be a fail instead of a skip.  If we're looking to move this test to conformance eventually, conformance tests can't skip